packages/lab/src/slider/internal/useKeyDownThumb.ts

```diff
@@ -0,0 +1,37 @@
+import { SliderChangeHandler, SliderValue } from "../types";
+import { roundToStep, roundToTwoDp } from "./utils";
+
+export function useKeyDownThumb(
+  min: number,
+  max: number,
+  step: number,
+  value: SliderValue,
+  setValue: SliderChangeHandler,
+  onChange: SliderChangeHandler | undefined
+) {
+  return (event: React.KeyboardEvent) => {
+    let valueItem: number = value;
+    switch (event.key) {
+      case "Home":
```

Comment:
maybe need a event.preventDefault() somewhere to stop home and end keys scrolling the whole page